_examples/mouse.go

```diff
@@ -19,7 +19,7 @@ func main() {
 	}
 	defer g.Close()
 
-	g.Cursor = true
+	g.Cursor = false
```

Comment:
Why did you make this change in the mouse example?

Reply:
Currently the display behaviour of the cursor is inconsistent between the different views (it doesn't display at all in `Button 2`), and since the example is demonstrating mouse control it seemed to be counter-intuitive to display the cursor
![image](https://user-images.githubusercontent.com/53288101/143493087-5d88a6d5-5cf5-4a01-a5fa-33d51fd37e03.png)

Reply:
That seems reasonable, thanks!